test/multiverse/suites/ruby_openai/ruby_openai_instrumentation_test.rb

```diff
@@ -68,8 +68,30 @@ def test_chat_completion_records_summary_event
     summary_events = events.filter { |event| event[0]['type'] == NewRelic::Agent::Llm::ChatCompletionSummary::EVENT_NAME }
 
     assert_equal 1, summary_events.length
+  end
 
-    # TODO: Write tests that validate the event has the correct attributes
+  def test_chat_completion_events_assign_all_attributes
+    in_transaction do
+      stub_post_request do
+        client.chat(parameters: chat_params)
+      end
+    end
+    _, events = @aggregator.harvest!
+    summary_events = events.filter { |event| event[0]['type'] == NewRelic::Agent::Llm::ChatCompletionSummary::EVENT_NAME }
+    attributes = summary_events[0][1]
+
+    refute_empty attributes['id']
+    refute_empty attributes['span_id']
+    refute_empty attributes['trace_id']
+    refute_empty attributes['response.model']
```

Comment:
Since this event is getting generated from the pre-built body. For the attributes where we feel pretty confident about the value (ex. 'gpt-3.5-turbo-0613' for 'response.model'), what do you think about checking the value as part of the assertion? 

```suggestion
    assert_equal 'gpt-3.5-turbo-0613', attributes['response.model']
```